mitodl/mitxonline
# Populate letter grades for 3T2022

# What are the relevant tickets?
Fix https://github.com/mitodl/mitxonline/issues/1647

# Description (What does it do?)
Populate letter grades for 3T2022


# How can this be tested?
Have a course run with a courseware id equal to any of the 5 DEDP course run in the Fall of 2022.
Create a user course run grade with letter grade set to 'Pass' and passed set to true.
Run the migration.
Check user's grade was converted to a letter.
